### PR TITLE
add page description and image to issues for FB links

### PR DESCRIPTION
**Pull request by PetrDlouhy**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class ApplicationController < ActionController::Base
   before_filter :no_disabled_users
   before_filter :set_auth_user
   before_filter :load_group_from_subdomain
+  before_filter :set_page_description
+  before_filter :set_page_image
```
Comment (nikolai-b) [inline app/controllers/application_controller.rb]: I don't think we need these

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -96,6 +98,24 @@ def load_group_from_subdomain
...
+    @page_description = description
+  end
+
+  def page_description
+    @page_description
+  end
+  helper_method :page_description
+
+  def set_page_image(description = nil)
+    @page_image = description
```
Comment (nikolai-b) [inline app/controllers/application_controller.rb]: Can you rename `description` -> `image`

File: app/controllers/issues_controller.rb [hunk 3]
```diff
@@ -16,6 +16,10 @@ def index
   def show
     @issue = IssueDecorator.decorate issue
     set_page_title @issue.title
+    set_page_description @issue.description
+    if @issue.photo
+       set_page_image @issue.photo.url
+    end
```
Comment (nikolai-b) [inline app/controllers/issues_controller.rb]:
```ruby
set_page_image @issue.photo.url if @issue.photo
```
or even
```ruby
set_page_image @issue.photo.try(:url)
```
are a bit neater

**Comment by nikolai-b** [comment]
Thanks again @PetrDlouhy!  I've made a few minor comments but it looks good to me

**Comment by PetrDlouhy** [comment]
The issues in comments were adjusted and tested.

**Pull request merged**